docs/tutorials/google/xeb_calibration_example.ipynb

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
<p>this cell doesn't work as written and requires an actual processor id. We should 1) open a bug to provide mock engine calibrations 2) in the mean time, wrap this in a conditional to skip if processor_id isn't supplied</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Agree on (1). For (2), I think this tutorial will look strange saying "look at calibration and pick good qubits" then not showing any calibration data. Do you think its sufficient to just save outputs for now, then follow-up after being unblocked by (1)? </p>

Reply:
<p>I agree that we can open a bug and fix this in a follow-up PR. For now, let's use the saved outputs. </p>